playbooks/deploy.yaml

```
@@ -51,9 +49,9 @@
       - name: candlepin
         password: "{{ candlepin_db_password }}"
       - name: foreman
-        password: "{{ foreman_db_password }}"
+        password: "{{ foreman_database_password }}"
       - name: pulp
```

We now have a variable for this:
```suggestion
      - name: "{{ pulp_database_user }}"
```